src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,50 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
```

Comment:
Yes, this was to show how to use the ducks pattern. It must be exported here for the saga.